plugins/destination/postgresql/client/list_tables.go

```diff
@@ -108,6 +116,7 @@ func (c *Client) listTables(ctx context.Context) (schema.Tables, error) {
 			Name:       columnName,
 			PrimaryKey: isPrimaryKey,
 			NotNull:    notNull,
+			Unique:     isUnique,
```

Comment:
I'm not sure if we should also set `Unique: true` when the column is a PK. I'm not sure it matters, but I think that if we set a column to both `PrimaryKey: true` and `Unique: true`, then create the relevant table, then try to migrate it, we will always load it as `PrimaryKey: true` and `Unique: false` so we'll detect a schema change

Reply:
Might worth checking other destinations too since we do have that change type now

Reply:
I am removing all of the changes for uniqueness and can handle this in a different PR